contract/models/account_anayltic_account.py

```diff
@@ -3,151 +3,47 @@
 # © 2014 Angel Moya <angel.moya@domatix.com>
```

Comment:
This file is misspelled. `s/account_anayltic_account.py/account_analytic_account.py`